client/components/repository/WorkflowBoard/Filters.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="d-flex align-center px-4">
+    <v-text-field
+      @input="updateFilter('searchText', $event)"
+      :value="searchText"
+      prepend-inner-icon="mdi-magnify"
+      placeholder="Search by ID or name"
+      clearable
+      class="search-field" />
+    <div v-if="assigneeOptions" class="ml-7 mr-3">
+      <assignee-avatar
+        v-for="{ id, isActive, ...assignee } in assigneeOptions"
+        :key="`assignee-${id}`"
+        @click="toggleAssignee(id)"
+        v-bind="assignee"
+        :class="[{ active: isActive }]"
+        class="avatar" />
+      <assignee-avatar
+        v-if="showUnassigned"
+        @click="updateFilter('unassigned', !unassigned)"
+        :class="{ active: unassigned }"
+        class="avatar" />
+    </div>
+    <v-btn
+      @click="updateFilter('recentOnly', !recentOnly)"
+      :class="{ active: recentOnly }"
+      text
+      class="btn-filters mx-1 text-capitalize">
+      Recently updated
+    </v-btn>
+  </div>
+</template>
+
+<script>
+import AssigneeAvatar from '@/components/repository/common/AssigneeAvatar';
+import xor from 'lodash/xor';
+
+export default {
+  name: 'workflow-board-filters',
+  props: {
+    assigneeOptions: { type: Object, default: () => ({}) },
+    showUnassigned: { type: Boolean, default: false },
+    searchText: { type: String, default: null },
+    assigneeIds: { type: Array, default: () => ([]) },
```

Comment:
Consider renaming this to selectedAssigneeIds and grouping assignee related props.

Reply:
`selectedAssigneeIds` is a better name, but it would prevent us from binding the whole filters state to the `Filters` component using `v-bind.sync="filters"` which seems nice. It also provides a context to the value's meaning.
What do we prefer?

Reply:
After quickly skimming the parent component, it seems to me that selectedAssigneeIds is a nicer name there as well. If I am missing something just ignore the comment and keep it this way 😉